DoLah/Commands/Command.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include "..\Models\Task.h"
+
+namespace DoLah {
+
+	class Calendar;
+
+	class AbstractCommand {
```

Comment:
AbstractTask is an abstract class and it is different from ITaskCommand which should be treated as solely an interface, even though C++ doesn't enforce virtual implementation.